frontend/tests/acceptance/gitlab-test.js

```diff
@@ -0,0 +1,220 @@
+import { module, test } from 'qunit'
+import { setupApplicationTest } from 'ember-qunit'
+import {
+  authenticateSession,
+  invalidateSession
+} from 'customer-center/tests/helpers/ember-simple-auth'
+import startApp from '../helpers/start-app'
+import destroyApp from '../helpers/destroy-app'
+import { selectChoose, clearSelected } from 'ember-power-select/test-support'
+import { clickTrigger } from 'ember-basic-dropdown/test-support/helpers'
+import { calendarSelect } from 'ember-power-calendar/test-support'
+import moment from 'moment'
+
+module('Acceptance | GitLab', function(hooks) {
+  setupApplicationTest(hooks)
+
+  hooks.beforeEach(function() {
+    this.application = null
+  })
+
+  hooks.beforeEach(async function() {
+    this.application = startApp()
```

Comment:
Isn't this done by `setupApplicationTest`?

Reply:
And why do you have two times `hooks.beforeEach`?

Reply:
no setupApplicationTest does not setup mirage in the version i have installed. the testing will be updated to the latest qunit and mirage in #71